packages/dashboard/package.json

```diff
@@ -38,8 +38,8 @@
     "json-stable-stringify": "^1.0.1",
     "react-avatar-editor": "^12.0.0",
     "react-hook-form": "^7.22.5",
-    "react-router": "^6.0.0-beta.0",
-    "react-router-dom": "^6.0.0-beta.0",
+    "react-router": "6.0.0-beta.0",
```

Comment:
https://github.com/DimensionDev/Maskbook/blob/d88e6eb5071cf69689cb54bfdc5b3231e8c82073/packages/dashboard/src/pages/routes.tsx#L21-L25 will fails at `react-router{,-dom}@6.1`, this is a temporary walkaround.